aws/resource_aws_dx_connection.go

```diff
@@ -0,0 +1,116 @@
+package aws
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/directconnect"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsDxConnection() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsDxConnectionCreate,
+		Read:   resourceAwsDxConnectionRead,
+		Delete: resourceAwsDxConnectionDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"band_width": &schema.Schema{
```

Comment:
nitpick, but I never saw this spelled as "band width" and the API also uses 1 word - bandwidth - mind changing that?